app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is a list of all the users</h1>
+<ul class="w-1/4 flex flex-col gap-4">
+  <% @users.each do |user| %>
+    <li class="flex items-center gap-2 border-2 p-4">
+      <img class="w-24 h-24" src=<%= user.photo %> alt=<%= user.name %> />
+      <div>
+        <p>Name: <%= user.name %></p>
+        <p>Number of posts: <%= user.posts_counter %></p>
+      </div>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] A number of posts is wrong. The user has 2 posts that ı create but it shows 6. Kindly fix it.
- [ ] Also ı should be able to see the button on this page. And with a click on the button ı should see all posts. Kindly fix it. [Wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png)
![image](https://user-images.githubusercontent.com/38635158/185726340-c4a62233-008d-441b-af7d-5c0cd6cb7f45.png)

Reply:
This requirement I believe is for the next milestone when we have to create forms.

Reply:
@jssol Number of posts or number of comments automatically should come from methods that you already create it in old milestone.